python/doc/examples/graphs/plot_graphs_loglikelihood_contour.py

```diff
@@ -171,21 +171,22 @@ def logLikelihood(X):
 # levels, while others are very different.
 # Anyway, it is obviously nicer to get a colored graphics.
```

Comment:
The section title "Multicolor contour plot" and the introduction are not suited to the actual graphs.
I would replace the section title with "Grayscale contour plot" and remove the introductory paragraph.